recipes/bamhash/1.1/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: bamhash
+  version: 1.1
+
+source:
+  url: https://github.com/DecodeGenetics/BamHash/archive/1c987784929f5c6cf28b9d36d00017380b922c71.tar.gz
+
+requirements:
+  build:
+    - gcc # [linux]
+    - openssl
+    - zlib
+
+  run:
+    - openssl
+    - libgcc
+    - zlib
+
+test:
+  commands:
+    - 'bamhash_checksum_bam --version'
+    - 'bamhash_checksum_fastq --version'
+
```

Comment:
Ah, sorry. I didn't get that.

Here I added `bam_checksum_fasta `:
`+ /bin/bash -x -e /home/hassan/anaconda3/conda-bld/test-tmp_dir/run_test.sh
+ bamhash_checksum_bam --version
bamhash_checksum_bam version 1.1
+ bamhash_checksum_fastq --version
bamhash_checksum_fastq version 1.1
+ bam_checksum_fasta --version
/home/hassan/anaconda3/conda-bld/test-tmp_dir/run_test.sh: line 5: bam_checksum_fasta: command not found
TESTS FAILED: bamhash-1.1-0
`

But I think @dpryan79 meant `bamhash_checksum_fasta`, which also fails:

`+ /bin/bash -x -e /home/hassan/anaconda3/conda-bld/test-tmp_dir/run_test.sh
+ bamhash_checksum_bam --version
bamhash_checksum_bam version 1.1
+ bamhash_checksum_fastq --version
bamhash_checksum_fastq version 1.1
+ bamhash_checksum_fasta --version
/home/hassan/anaconda3/conda-bld/test-tmp_dir/run_test.sh: line 5: bamhash_checksum_fasta: command not found
TESTS FAILED: bamhash-1.1-0
`